ui/components/ui/logo/logo-onekey.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+type LogoOneKeyType = {
+  width: string;
+  color: string;
+  className: string;
+  ariaLabel: string;
+};
+
+const LogoOnekey = ({
```

Comment:
Very nit: But this should be `LogoOneKey` (with a capitalized `K`), for consistency